rewards/classes/RewardsManager.py

```diff
@@ -44,29 +51,30 @@ def __init__(self, chain: Network, cycle: int, start: int, end: int, boosts):
         self.apy_boosts = {}
 
     def fetch_sett_snapshot(
-        self, block: int, sett: str, blacklist: bool = True
+        self, start_block: int, end_block: int, sett: str, blacklist: bool = True
     ) -> Snapshot:
-        return sett_snapshot(self.chain, block, sett, blacklist)
```

Comment:
@mitche50 - this is the most important change here. Instead of taking one snapsho, we now take multiple snaps based on NUMBER_OF_HISTORICAL_SNAPSHOTS_FOR_SETT_REWARDS const and sum up balances.
Also, summed up balances will be boosted by multiplier